app/controllers/authors_controller.rb

```diff
@@ -9,4 +9,17 @@ def show
 
   def index
   end
+
+  def destroy
+    relevant_book_authors_ids = BookAuthor.where(author_id: params[:id].to_i).pluck(:id)
```

Comment:
For MVC we should be doing the Where and pluck call in the model.

Reply:
Done - moved!